modules/chapter3/pages/section1.adoc

```diff
@@ -1,3 +1,68 @@
 = Section 1
 
-This is _Section 1_ of _Chapter 3_ in the *hello* quick course....
\ No newline at end of file
+== AI Pipelines
```

Comment:
This is a nice introductory overview of pipelines and Elyra, which is best placed in chapter 1, the theoretical background (see [design doc](https://docs.google.com/document/d/1nIfEaLDELWMC24Q9GlS19hQIQba-bOmhAiIjomsyU8Y/edit#heading=h.dfpktd2cs6qg)).

Please coordinate with @codrinbucur to move the introductory sub sections to chapter 1.